NOTICE.txt

```diff
@@ -1,314 +1,135 @@
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
+	Homepage:	http://www.hazelcast.com/hazelcast/
 
-This product depends on Proxy Service 1.0
+This product depends on hazelcast-aws 1.1.1
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
-	Homepage:	https://github.com/kaazing/gateway.service.proxy.git
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
+	Homepage:	http://www.hazelcast.com/
 
-This product depends on gateway.service.update.check.management 1.0
+This product depends on hazelcast-client 3.7.4
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
-	Homepage:	https://github.com/kaazing/gateway.service.update.check.management.git
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
+	Homepage:	http://www.hazelcast.com/hazelcast-client/
 
-This product depends on Update Check Service 1.0
+This product depends on Commons CLI 1.2
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
-	Homepage:	https://github.com/kaazing/gateway.service.update.check.git
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
+	Homepage:	http://commons.apache.org/cli/
 
-This product depends on Gateway Service 1.0
+This product depends on Commons IO 2.4
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
-	Homepage:	https://github.com/kaazing/gateway.service.git
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
+	Homepage:	http://commons.apache.org/io/
 
-This product depends on BIO Transport 1.0
+This product depends on Java Unified Expression Language API 2.2.7
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
-	Homepage:	https://github.com/kaazing/gateway.transport.bio.git
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
+	Homepage:	http://juel.sourceforge.net/juel-api/
 
-This product depends on Http Transport 1.0
+This product depends on Java Unified Expression Language Implementation 2.2.7
 
-	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
-	Homepage:	https://github.com/kaazing/gateway.transport.http.git
+	License:	http://www.apache.org/licenses/LICENSE-2.0.txt (The Apache Software License, Version 2.0)
+	Homepage:	http://juel.sourceforge.net/juel-impl/
 
-This product depends on NIO Transport 1.0
+This product depends on Netty 3.10.5.Final
 
 	License:	http://www.apache.org/licenses/LICENSE-2.0 (Apache License, Version 2.0)
-	Homepage:	https://github.com/kaazing/gateway.transport.nio.git
+	Homepage:	http://netty.io/
 
-This product depends on Pipe Transport 1.0
+This product depends on Javassist 3.7.ga
```

Comment:
This is a runtime dependency, is it possible to exclude those as we don't depend on them